sortingview/experimental/SpikeSortingView/MultiTimeseries.py

```diff
@@ -0,0 +1,40 @@
+import os
+from typing import List, Union
+import kachery_client as kc
+from .Figure import Figure
+
+
+class MultiTimeseries:
+    def __init__(self, *, label: str) -> None:
+        self._label = label
+        self._panels = []
+    def add_panel(self, figure: Figure, *, relative_height: Union[float, None]=None):
+        self._panels.append({
+            'figure': figure,
+            'relative_height': relative_height
+        })
+    def get_composite_figure(self):
+        if not os.getenv('FIGURL_CHANNEL'):
+            raise Exception('Environment variable not set: FIGURL_CHANNEL')
+        panels = []
+        for p in self._panels:
+            fig0: Figure = p['figure']
+            v = {
+                'type': fig0.data['type'],
+                'label': fig0.label,
+                'figureDataSha1': _upload_data_and_return_sha1(fig0.get_serialized_figure_data())
+            }
+            if p['relative_height'] is not None:
+                v['relativeHeight'] = p['relative_height']
+            panels.append(v)    
```

Comment:
So the distinction here is that each of the `Figure`s in the `_panels` collection could be a stand-alone time series figure, but when they've been collected as a `MultiTimeseries` that's a new kind of figure that has special rules on the front-end.

Each `v` ("view" is the mnemonic here?) is taking a could-have-been-viewed-independently `Figure` and extracting the type, label, and data, along with possibly a relative height, and then adding them to the `panels` collection. Understanding the context, this all makes sense.

My only question--is there a benefit to storing the full `Figure` data in `self._panels` and then extracting a subset of it in `get_composite_figure` vs storing only the type, label, sha1, and (optional) relative height to begin with?

Reply:
> My only question--is there a benefit to storing the full `Figure` data in `self._panels` and then extracting a subset of it in `get_composite_figure` vs storing only the type, label, sha1, and (optional) relative height to begin with?

This just provides a simpler interface. You use the same figure object to view a standalone figure as to build a composite figure.